packages/desktop/views/dashboard/collectibles/views/CollectiblesGalleryView.svelte

```diff
@@ -1,8 +1,18 @@
 <script lang="ts">
     import { IllustrationEnum } from '@auxiliary/illustration'
+    import { openPopup, PopupId } from '@auxiliary/popup'
     import { localize } from '@core/i18n'
     import { nftSearchTerm, queriedNfts, ownedNfts } from '@core/nfts'
-    import { FontWeight, Illustration, Text, ReceiveButton, SearchInput, NftGalleryItem } from 'shared/components'
+    import { FontWeight, Illustration, Text, SearchInput, NftGalleryItem, Button, ButtonSize } from 'shared/components'
+
+    function handleReceiveFundsPopup(): void {
+        openPopup({
+            id: PopupId.ReceiveAddress,
+            props: {
+                title: localize('general.receiveFunds'),
```

Comment:
`localize('general.receiveFunds')` is the default in the component, but the text that was being sent from this component was `localize('actions.depositNft')`
It makes more sense to me `depositNft`, wdyt?